src/player-interaction.js

```diff
@@ -132,12 +133,15 @@ class PlayerInteraction {
     if (player.hasOption) {
       availableActions.push('check');
       availableActions.push('raise');
+      availableActions.push('all-in');
     } else if (hasBet) {
       availableActions.push('call');
       availableActions.push('raise');
+      availableActions.push('all-in');
     } else {
       availableActions.push('check');
       availableActions.push('bet');
+      availableActions.push('all-in');
     }
 
     // Prevent players from raising when they don't have enough chips.
```

Comment:
We should also remove the all-in action when players only have enough chips to call, just like we do with raise (it could be made more explicit with "Call (All-in)."